From 0c62106efd92377d3a37d8511fc1210ccee6d0ac Mon Sep 17 00:00:00 2001
From: Patrice Chotard <patrice.chotard@foss.st.com>
Date: Thu, 21 Dec 2023 14:19:51 +0100
Subject: [PATCH 0880/1141] memory: stm32: Remove unused mm_phys_base field
 from stm32_omi struct

Remove unused mm_phys_base field from stm32_omi struct.

Change-Id: I4074d9077cfd457dec23e1dcbb00aa407281f3eb
Signed-off-by: Patrice Chotard <patrice.chotard@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/348418
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Domain-Review: Christophe KERELLO <christophe.kerello@st.com>
Reviewed-by: Christophe KERELLO <christophe.kerello@st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
---
 drivers/memory/stm32-omi.c | 1 -
 include/memory/stm32-omi.h | 1 -
 2 files changed, 2 deletions(-)

diff --git a/drivers/memory/stm32-omi.c b/drivers/memory/stm32-omi.c
index 9fda9ab80012..ea052290fb20 100644
--- a/drivers/memory/stm32-omi.c
+++ b/drivers/memory/stm32-omi.c
@@ -558,7 +558,6 @@ static int stm32_omi_probe(struct platform_device *pdev)
 	of_node_put(node);
 
 	if (rmem) {
-		omi->mm_phys_base = rmem->base;
 		omi->mm_size = rmem->size;
 		omi->mm_base = devm_ioremap(dev, rmem->base, rmem->size);
 		if (IS_ERR(omi->mm_base)) {
diff --git a/include/memory/stm32-omi.h b/include/memory/stm32-omi.h
index 5799e3640398..bd182567a384 100644
--- a/include/memory/stm32-omi.h
+++ b/include/memory/stm32-omi.h
@@ -167,7 +167,6 @@ struct stm32_omi {
 
 	void __iomem *regs_base;
 	void __iomem *mm_base;
-	phys_addr_t mm_phys_base;
 	phys_addr_t regs_phys_base;
 	resource_size_t mm_size;
 	u32 clk_rate;
-- 
2.39.2

